### ISSUE TITLE
Can't use CommandAPICommand Class for creating a command.

### ISSUE THREAD

**Issue by dhanotboi** [JorelAli/CommandAPI#226]
**CommandAPI version:** _6.0.0_

**Minecraft version:** _1.17.1_

**What I did:**

- I tried using the CommandAPICommand class as shown in the documentation for command registration.
    `new CommandAPICommand("broadcastmsg")
    .withArguments(new GreedyStringArgument("message"))
    .withAliases("broadcast", "broadcastmessage")
    .register();`

**Actual result:**

about 17 errors showed up.
![image](https://user-images.githubusercontent.com/44154103/123615637-12328100-d838-11eb-9585-4f17a9768575.png)
![image](https://user-images.githubusercontent.com/44154103/123615762-31311300-d838-11eb-9d81-c311f084cbf9.png)

**What should have happened:**

- I expected that it would work and not give me errors.
- I'm pretty sure this is based around the IDE thinking that new CommandAPICommand(), is a new method and not used from the import, I had `import dev.jorel.commandapi.CommandAPICommand;` so I do not understand why this is happening
-

**Comment by JorelAli** [comment]
@dhanotboi This is most likely due to an incomplete set up in your IDE. Have you added the dependency correctly, and if so can you confirm that there is no invalid code before you call `new CommandAPICommand()`?

The CommandAPI's documentation examples are compile-checked so I can confirm with 100% certainty that the examples do work!

**Comment by dhanotboi** [comment]
So I reloaded my IDE, and I have made sure my pom.xml was alright but the issue still occurs. It may be because of the fact that the IDE is assuming the import to be "useless" and asking me to remove it, however I don't know how to fix that. 
My code seems to be extremely simple so I wouldn't know why it would be caused because of that.
![image](https://user-images.githubusercontent.com/44154103/123622756-35146380-d83f-11eb-89a4-3a31e0602ade.png)
